refinery_migrations/Cargo.toml

```diff
@@ -34,3 +34,6 @@ tokio = { version = "0.2", features = ["full"], optional = true }
 
 [dev-dependencies]
 tempfile = "3.1.0"
+
+[package.metadata.docs.rs]
```

Comment:
`refinery_migrations` is part of `refinery` but is only used to use the proc_macros, shouldn't be used alone, do we need to publish it?

Reply:
I was actually going to ask about that. It looks like `refinery_macros` is used to expose the proc macros (which does need to be published). But can `refinery` and `refinery_migrations` be consolidated into a single crate? Currently, both are published but it might be a bit confusing for end users since `refinery` reexports most of `refinery_migrations`.

Reply:
yeah, it's definitely not optimal, i think the reason is that back when we started developing `refinery`, `refinery_cli` used functions from `refinery` that were not meant to be public,  since `refinery_cli` also has to be an independent crate , we moved them to `refinery_migrations`.
We have since pub'ed `migrate_from_config`, so the reason doesn't seem to be legit anymore.  
I took quick glance at refinery_cli and it seems the only function that is still used without being doc'ed and oficially pub'ed is `find_migration_filenames`, maybe we can also make it pub?

let's do this, for now let's remove that package meta from refinery_migrations, and merge this PR, i'll then open an issue to deprecate `refinery_migrations` and merge it's functions into `refinery`, what do you think?

Reply:
Sounds good, made the change

Reply:
thanks!